source/CMakeLists.txt

```diff
@@ -122,33 +115,8 @@ foreach(SAMPLE ${SAMPLES})
         target_include_directories(${SAMPLE_NAME} SYSTEM PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/3rd-party/clipp/include)
     endif()
 
-    add_dependencies(${SAMPLE_NAME} CopyZdf)
-
 endforeach()
 
 # TODO: Generalize how input file dependencies are copied, see issue #46
```

Comment:
Can remove this TODO, and mark #46 as closed.

Reply:
done